guide/foundations/backups.md

```diff
@@ -0,0 +1,265 @@
+   retina = "/assets/images/guide/foundations/backups/backups@2x.jpg"
+   mobile = "/assets/images/guide/foundations/backups/backups-mobile.jpg"
+   mobileRetina = "/assets/images/guide/foundations/backups/backups-mobile@2x.jpg"
+   alt-text = "Several devices exchanging data"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Bitcoin backups
+
+The self-custody of bitcoin is an important right, but also a financial responsibility that can take practice getting used to, and scale from pocket money to life savings.
+
+Making a secure backup of your wallet should be straightforward. This guide is meant to provide a starting point for simple bitcoin backups. From a basic setup, you can increase the security level as your funds grow. 
+
+
+## What are backups?
+
+When we talk about backups, we mean saving the information needed to restore access to a wallet outside of the application that created the private keys. Most often, this will be the *recovery-phrase* of of 12 or 24 words, but can also include other information like a *pass-phrase*.
+
+The main risks that backups need to protect against are:
+
+- Loss of wallet - you lost the device and application holding the private key
+- Remote theft - your private key got discovered online by a bad actor
+- Physical damage - your backup got destroyed
+- Theft on discovery - your backup got discovered in person by a bad actor
+- Catastrophic events - your backup location got destroyed
+- Inheritance loss - next-of-kin were unable to access your wallet
+
+
+## Backup levels, basic to advanced
+
+A basic backup can be done in a few minutes, while more advanced setups can require the purchase of dedicated material and more effort. 
+
+Let’s look at some real life examples of backup methods for single-key wallets and what they protect against. (Multi-key setups require their own writeup.)
+
+<br>
+
+{% include image.html
+   image = "/assets/images/guide/foundations/backups/backup-table.png"
+   retina = "/assets/images/guide/foundations/backups/backup-table@2x.png"
+   alt-text = "Table of backup levels"
+%}
+
+<br>
+
+#### Level 0
+## Encrypted cloud backup 
+
+By storing the recovery-phrase in an online location that is encrypted (not in plain text) and hard to access by anyone other than yourself, you are protected against *loss of wallet*. This is the absolute minimum you should do and only takes a few minutes.
+
+{% include image.html
+   image = "/assets/images/guide/foundations/backups/cloudbackup.png"
+   retina = "/assets/images/guide/foundations/backups/cloudbackup@2x.png"
+   alt-text = "Encrypted cloud backup"
+%}
+
+**Do**
+- Use an encrypted password manager like 1Password, LastPass, iCloud Keychain
+- Alternatively, use a wallet application with automatic cloud backup
+
+**Don’t** 
+- Screenshot the recovery-phrase and save it in Google Photos, iCloud photos
+- Write down the recovery-phrase in plain text on your device, Google Docs, Dropbox, Notes etc.
```

Comment:
I'm a bit confused, this now talks to the `end-user` not really an app designer or dev, so they should discourage the screenshots, etc.

Reply:
As per discussions on sprint planning call, we will keep this page talking to the end user for now.